views/layout.erb

```diff
@@ -10,10 +10,13 @@
     <link rel="stylesheet" href="/stylesheets/master.css" type="text/css" />
     <link rel="shortcut icon" href="/favicon.ico" />
     <%= @html_head %>
-    <script src="/scripts/head.load.min.js"></script>
-    <script>
```

Comment:
Please retain the `@javascripts` variable. You can replace the `head.js` call with a for loop a la
```
<% @javascripts.each do |js| %>
  <script src="<%= js %>"></script>
<% end %>
```